.github/workflows/build-pull-request.yml

```diff
@@ -59,35 +75,3 @@ jobs:
             esac
           done
 
-  build:
```

Comment:
This step needs to be there for CI to work. 

Find a way to get `$path2` over here.